Content.Server/Atmos/EntitySystems/GasTankSystem.cs

```diff
@@ -359,7 +359,8 @@ public void CheckStatus(Entity<GasTankComponent> ent)
         /// </summary>
         private void OnAnalyzed(EntityUid uid, GasTankComponent component, GasAnalyzerScanEvent args)
         {
-            args.GasMixtures = new Dictionary<string, GasMixture?> { {Name(uid), component.Air} };
+            args.GasMixtures ??= new List<(string, GasMixture?)>();
```

Comment:
Why are these assignments null-coalescing? Can `OnAnalyzed` now be called multiple times?

Reply:
I asked @ElectroJr about this over Discord. This was changed by him in #26493, but only in GasFilterSystem.cs. He said the other files should be doing the same, so that the list does not get overwritten. His explanation was "The reason for changing it was just so that it actually supports scanning gases from an entity with multiple gas containing components". Im am not that familiar with how events work, but I think this is not really needed, but should it support this now in case more entities are added. At least this should be done in a consistent way in all OnAnalyzed functions, which it now does.

Reply:
Understood, thanks.